ingestion/src/metadata/ingestion/sink/metadata_rest.py

```diff
@@ -159,9 +162,13 @@ def write_tables(self, db_and_table: OMetaDatabaseAndTable):
 
             created_table = self.metadata.create_or_update(table_request)
             if db_and_table.location is not None:
+                if db_and_table.location.description is not None:
+                    db_and_table.location.description = (
+                        db_and_table.location.description.strip()
+                    )
```

Comment:
Why is it between parentheses

Reply:
its the code formatter moving this into next line

Reply:
<img width="1129" alt="image" src="https://user-images.githubusercontent.com/38649/146440822-4a83a25e-4dac-4383-b2d5-6169e055b167.png">

Reply:
👍